docs/index.md

```diff
@@ -193,3 +197,7 @@ echo callPardotApi('https://pi.pardot.com/api/login/version/3',
     )
 );
 ```
+
+## Supported API wrappers
+
+* [ruby-pardot](https://www.github.com/Pardot/ruby-pardot)
```

Comment:
Looks like we only have one official API wrapper? If we have more, I will add them.